### PR TITLE
aws-ofi-nccl: Add 1.8.1 to 1.13.0

### PR DESCRIPTION
**Pull request by msimberg**

### CODE REVIEW

**Review by albestro — APPROVED** [review]
✅ sha verified

File: var/spack/repos/builtin/packages/aws-ofi-nccl/package.py [hunk 1]
```diff
@@ -18,6 +18,16 @@ class AwsOfiNccl(AutotoolsPackage):
...
+    version("1.13.0", sha256="50dd231a0a99cec29300df46b8e828139ced15322a3c3c41b1d22dcc9a62ec02")
+    version("1.12.1", sha256="821f0929c016e5448785bbc6795af5096559ecfc6c9479eb3818cafa61424576")
+    version("1.12.0", sha256="93029207103b75f4dc15f023b3b8692851202b52b7e2824723dd5d328f0ea65b")
+    version("1.11.1", sha256="a300e620e03ba3cc0915f9d466232ff0bf6c84edf4e2cd93592d53cf2a62741b")
+    version("1.11.0", sha256="45d935133b183c945c16b70d8428d676a554faf5bd922b7909e9f1ec55ba6168")
+    version("1.10.0", sha256="ed63f627b42c7b0f7312ce2916a3c4dfeb5145f78b492c0d1e0d0a6828a0474c")
+    version("1.9.2", sha256="f763771e511ae3bc7bb708795f9802867a4a2bc4e4df6a265c7f6a033e9a8b9a")
+    version("1.9.1", sha256="3ee01258674e70d6966eb6d319461f9b882afae618e217e0ae7ec03d26169b35")
+    version("1.9.0", sha256="8d6d0469110a89b5431836d263860fb60fde7beccb26f553de41dca1feb61b51")
+    version("1.8.1", sha256="beb59959be0f60b891f9549f4df51b394e97e739416c88c3436e75516fe067c8")
```
Comment (aurianer) [inline var/spack/repos/builtin/packages/aws-ofi-nccl/package.py:30]: seems a duplicate

Comment (albestro) [reply var/spack/repos/builtin/packages/aws-ofi-nccl/package.py:30]: you're absolutely right, I didn't notice it! @aurianer do you want to open a PR fixing that?

Comment (aurianer) [reply var/spack/repos/builtin/packages/aws-ofi-nccl/package.py:30]: #47880

**Pull request merged**